yhondanilo/vocales2.rb

```diff
@@ -0,0 +1,31 @@
+class Vowels
+
+  def ejecutar
+    tiempo_inicial = Time.now
+    search
+    tiempo_final = Time.now
+    puts "Se encontraron las siguientes vocales #{@char_count}" + "en un tiempo de #{ tiempo_final - tiempo_inicial }"
+  end
+
+  def search
+    File.open('text.txt','r') do |f|
```

Comment:
Space missing after comma.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.